src/main/resources/com/adobe/epubcheck/schema/30/mod/epub-mathml3.rnc

```diff
@@ -96,6 +82,6 @@ epub.annotation-xml =
           }
         epub.att-name.mathml.presentation =
           attribute name {
-            string "contentequiv"
+            xsd:NCName
```

Comment:
name is only defined as expecting a [string value](https://www.w3.org/TR/MathML/chapter5.html#id.5.2.3.2).

Reply:
yeah, I copied the definition from from the normative [MathML schemas](https://www.w3.org/TR/MathML/appendixa.html#parsing_name).
If I understand correctly, `name` represents a symbol in a [Content Dictionary](https://www.w3.org/TR/MathML/chapter4.html#contm.cds), and its syntax (along with the companion attribute `cd`) is defined in the [`csymbol` element](https://www.w3.org/TR/MathML/chapter4.html#contm.csymbol), where it seems to make sense for it to be an `NCName`…

Anyways, I suppose we can safely align with the MathML schemas there…

Reply:
Ya, it's weird that no mention is made why it can't have colons, but the csymbol definition points to the schema for its value definition and its ncname, so whatever...